eng/helix/content/RunTests/TestRunner.cs

```diff
@@ -247,7 +247,7 @@ public async Task<int> RunTestsAsync()
             {
                 // Timeout test run 5 minutes before the Helix job would timeout
                 var cts = new CancellationTokenSource(Options.Timeout.Subtract(TimeSpan.FromMinutes(5)));
-                var commonTestArgs = $"vstest {Options.Target} --logger:xunit --logger:\"console;verbosity=normal\" --blame";
+                var commonTestArgs = $"test {Options.Target} --logger:xunit --logger:\"console;verbosity=normal\" --blame \"CollectHangDump;TestTimeout=5m\"";                
```

Comment:
Oops, doesn't `vstest` expect a project or solution file❔ I'm hoping there's no need to change the subcommand.

Reply:
Note these options aren't particularly well-documented for either `dotnet test` or `dotnet vstest`. Might be more correct to use `--blame  --blame-hang  --blame-hang-timeout 5m` but it's hard to tell.

Reply:
thoughts @nohwnd as he's the expert on these new options